
joints_update:
  type: joint_state_controller/JointStateController
  publish_rate: 50
JointA_PositionController:
  type: position_controllers/JointPositionController
  joint: JointA
  pid: {p: 100.0, i: 0.0, d: 0.0}  # Optional: PID gains
  publish_rate: 50  # Optional: Control loop rate
  hardwareInterface: hardware_interface/PositionJointInterface
JointB_PositionController:
  type: position_controllers/JointPositionController
  joint: JointB
JointC_PositionController:
  type: position_controllers/JointPositionController
  joint: JointC
  # joints_update:
  #   type: joint_state_controller/JointStateController
  #   publish_rate: 50
  # position_controller:
  #   type: position_controllers/JointPositionController
  #   joint: Mot_1
  #   pid: {p: 1.0, i: 0.1, d: 0.01}
  # delta_controller:
  #   type: position_controllers/JointTrajectoryController
  #   joints:
  #     - Mot_1
  #     - Mot_2
  #     - Mot_3